File: src/main/java/de/sean/splugin/spigot/inventories/FriendAddInventory.java
package de.sean.splugin.spigot.inventories;

import org.bukkit.Bukkit;
import org.bukkit.inventory.Inventory;

public class FriendAddInventory {
    public static final String INVENTORY_NAME = "Add Friend";
    /// The content needs to be added when opened.
    public static Inventory inventory = Bukkit.createInventory(null, 9 * 3, INVENTORY_NAME);
}

File: BlockProt/src/main/kotlin/de/sean/blockprot/bukkit/inventories/FriendAddInventory.kt
package de.sean.blockprot.bukkit.inventories

import org.bukkit.Bukkit

object FriendAddInventory {
    const val INVENTORY_NAME = "Add Friend"
    fun createInventory() = Bukkit.createInventory(null, 9 * 3, FriendAddInventory.INVENTORY_NAME)
}
